quasiquotes3/shared/src/test/scala/scala/meta/tests/SuccessSuite.scala

```diff
@@ -0,0 +1,2955 @@
+package scala.meta.tests
+package quasiquotes
+
+import munit._
+import scala.meta._
+import scala.meta.dialects.Scala211
+import scala.meta.quasiquotes._
+import munit.Clue.generate
+import scala.meta.trees._
+import meta.prettyprinters.XtensionStructure
+
+import compat.Platform.EOL
+
+import meta.prettyprinters.XtensionSyntax
+
+class Success extends TreeSuiteBase {
+  test("rank-0 liftables") {
```

Comment:
Isn't this the same test suite like for Scala 2? Could we reuse it? Or would it need my PR first I guess?

Reply:
For now it has tests slightly changed to accommodate the `Any` types returned from the unapply. I hope to use the old suite once we can get `transparent inline unapply` working, outside of those 2 tests with non-standard type ascriptions of course